src/posts/announce-2.3.0.Final.adoc

```diff
@@ -0,0 +1,185 @@
+* [https://issues.jboss.org/browse/THORN-2232[THORN-2232]] Remove Teiid and its related Fractions from main code repo of Thorntail (Done)
+* [https://issues.jboss.org/browse/THORN-2233[THORN-2233]] improve formatting of artifact sizes in the fraction plugin output (Done)
+* [https://issues.jboss.org/browse/THORN-2234[THORN-2234]] make sure the fraction plugin doesn't modify the XML namespace in module.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2235[THORN-2235]] make sure the fraction plugin fills in the version attribute of the <module> root element in module.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2236[THORN-2236]] make sure the fraction plugin doesn't fail when computing fraction size and artifacts are missing locally (Done)
+* [https://issues.jboss.org/browse/THORN-2237[THORN-2237]] required updates to the fraction plugin (Done)
+* [https://issues.jboss.org/browse/THORN-2238[THORN-2238]] improve error handling in the fraction plugin (Done)
+* [https://issues.jboss.org/browse/THORN-2239[THORN-2239]] Create a new ee-security fraction (Done)
+* [https://issues.jboss.org/browse/THORN-2240[THORN-2240]] Update Wildfly Config API to generate an ee-security subsystem code (Done)
+* [https://issues.jboss.org/browse/THORN-2241[THORN-2241]] fix build so that all artifacts are properly built and SNAPSHOTs don't have to be downloaded when generating BOMs (Done)
+* [https://issues.jboss.org/browse/THORN-2249[THORN-2249]] make sure `mvn clean install` passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2250[THORN-2250]] make sure the standalone test suite passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2265[THORN-2265]] remove all usages of deprecated APIs after migration to WildFly 14 (Done)
+* [https://issues.jboss.org/browse/THORN-2269[THORN-2269]] the fraction plugin should allow sharing one module-rewrite.conf among multiple fractions (Done)
+* [https://issues.jboss.org/browse/THORN-2270[THORN-2270]] the fraction plugin should be able to remove an <artifact> from module.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2271[THORN-2271]] add workaround for WFLY-11373 to our integration of SmallRye Fault Tolerance (Done)
+* [https://issues.jboss.org/browse/THORN-2272[THORN-2272]] fix how Infinispan caches are forced to be active (Done)
+* [https://issues.jboss.org/browse/THORN-2273[THORN-2273]] don't let RESTEasy provide their own MicroProfile RestClient (Done)
+* [https://issues.jboss.org/browse/THORN-2274[THORN-2274]] upgrade Keycloak to 4.6.0.Final (Done)
+* [https://issues.jboss.org/browse/THORN-2275[THORN-2275]] make sure `mvn clean install` in the Examples passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2282[THORN-2282]] upgrade Keycloak to 4.7.0.Final (Done)
+* [https://issues.jboss.org/browse/THORN-2294[THORN-2294]] upgrade Keycloak to 4.8.1.Final (Done)
```

Comment:
As there's a LOT of issues here, it might be worth culling some intermediary ones. For instance there are four issues about updating Keycloak, when really the one updating to the most recent version is the only one that matters

Reply:
I left the one where Ladislav did upgrade to 4.5.0 as he did a number of related enhancements and the very last one, 4.8.1